RELEASE.md

```diff
@@ -1,3 +1,16 @@
+# Release 2.0.0-beta1
+
+Tensorflow 2.0.0-beta1 is a minor update to 2.0.0-beta0 with a few important bug
+fixes. Please refer to [2.0.0-beta0 release
+notes](https://github.com/tensorflow/tensorflow/releases/tag/v2.0.0-beta0) for a
+complete list of changes in 2.0.0-beta0.
+
+## Bug Fixes and Other Changes
+* Partially fix the function inlining and performance regression for LSTM/GRU.
```

Comment:
Partially?

Reply:
There is a minor use case for masked input, which is still using the normal low performant kernel and need to be addressed.